src/main/java/athena/commands/ListCommand.java

```diff
@@ -22,7 +24,7 @@ public ListCommand(String importance) {
      */
     @Override
     public void execute(TaskList taskList, Ui ui) {
-        TaskList tasksFiltered = taskList.getFilteredList(taskImportance);
+        TaskList tasksFiltered = taskList.getFilteredList(taskImportance, taskForecast);
```

Comment:
Adding this parameter will cause the program to not compile. We can leave this out for now.